sdk/eventhubs/azure-messaging-eventhubs/inc/azure/messaging/eventhubs/eventhubs_exception.hpp

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <azure/core/amqp/models/amqp_error.hpp>
+
+#include <stdexcept>
+#include <string>
+
+namespace Azure { namespace Messaging { namespace EventHubs {
+
+  enum class EventHubsStatusCode : int32_t
+  {
+    Invalid,
+    Ok,
+    Error,
+    Timeout,
+    Cancelled,
```

Comment:
Do we want 0 to be the invalid value?

Reply:
By tradition, the 0th entry in an enumeration should be an invalid value, unless you have a strong reason for it to be something else. That way you cannot accidentally treat a default initialized value as meaningful.

Having said that, this enumeration was unused.